identity-example/src/main/java/com/stripe/android/identity/example/network.kt

```diff
@@ -0,0 +1,31 @@
+package com.stripe.android.identity.example
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class VerificationSessionCreationResponse(
+    @SerialName("client_secret") val clientSecret: String,
+    @SerialName("ephemeral_key_secret") val ephemeralKeySecret: String,
+    @SerialName("id") val verificationSessionId: String,
+    @SerialName("url") val url: String
+)
+
+@Serializable
+data class VerificationSessionCreationRequest(
+    @SerialName("options") val options: Options? = null,
+    @SerialName("type") val type: String = "document"
+) {
+    @Serializable
+    data class Options(
+        @SerialName("document") val document: Document? = null
+    )
+
+    @Serializable
+    data class Document(
+        @SerialName("allowed_types") val allowedTypes: List<String>? = null,
```

Comment:
Do you need all the `@SerialName` if they have the same name?

Reply:
No, they can be automatically with same names. Will still keep them  here though in case we change the param on the server later